ignite/pkg/cosmosgen/generate_typescript.go

```diff
@@ -77,49 +61,52 @@ func (g *generator) generateTS(ctx context.Context) error {
 }
 
 func (g *tsGenerator) generateModuleTemplates(ctx context.Context) error {
-	gg := &errgroup.Group{}
 	dirCache := cache.New[[]byte](g.g.cacheStorage, dirchangeCacheNamespace)
-	add := func(sourcePath string, modules []module.Module) {
-		for _, m := range modules {
-			gg.Go(func() error {
```

Comment:
Running in the background can increase the time, as far as I remember, this logic inside can take some time

Reply:
i can add it back yes, this was helpful to remove while debugging.